guide.md

```diff
@@ -9,6 +9,10 @@ rapidly changing. A more detailed user guide will be published here when gittuf
 reaches beta. For now, this guide presents the workflow for using gittuf's alpha
 releases.
 
+## Prerequisites
+
+Before using gittuf, we suggest having a valid signing key specified in your git configuration (i.e. `git config --local user.signingkey`). Note that this is required for any gittuf operations that write changes to the repository.
```

Comment:
Can you reformat to 80 cols?

Reply:
Also, drop a link to https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work?

Reply:
As well as gitsign.dev.